pepdbagent/pepdbagent.py

```diff
@@ -374,71 +374,6 @@ def get_projects_in_namespace(
 
         return result_list
 
-    def get_projects_in_list(
-        self,
-        registry_paths: list,
-    ) -> List[peppy.Project]:
-        """
-        Get a list of projects as peppy.Project instances.
-        Get a list of projects in a list of registry_paths
-        :registry_paths: list with registry paths of the projects
-        :return: a list of peppy.Project instances for the requested projects.
-        """
-        if not isinstance(registry_paths, list):
-            raise TypeError(f"incorrect variable type provided")
-        for rpath in registry_paths:
-            if not is_valid_resgistry_path(rpath):
-                # should we raise an error or just warn with the logger?
-                raise ValueError(f"Invalid registry path supplied: '{rpath}'")
-
-        parametrized_filter = ""
-        for i in range(len(registry_paths)):
-            parametrized_filter += "(namespace=%s and name=%s)"
-            if i < len(registry_paths) - 1:
-                parametrized_filter += " or "
-
-        sql_q = f"select {NAME_COL}, {PROJ_COL} from {DB_TABLE_NAME} where {parametrized_filter}"
-        flattened_registries = tuple(
-            chain(
-                *[
-                    [r["namespace"], r["item"]]
-                    for r in map(
-                        lambda rpath: ubiquerg.parse_registry_path(rpath),
-                        registry_paths,
-                    )
-                ]
-            )
-        )
-        results = self.run_sql_fetchall(sql_q, *flattened_registries)
-
-        # extract out the project config dictionary from the query
-        return [peppy.Project(project_dict=p[1]) for p in results]
-
-    def get_all_projects(
```

Comment:
This will break pephub, but I believe this is a good choice - we shouldn't let people just get all projects in the database as you mentioned.